src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonSynchronisationService.kt

```diff
@@ -313,8 +313,27 @@ class ContactPersonSynchronisationService(
     }
   }
 
+  private suspend fun createPersonMapping(
+    mapping: PersonMappingDto,
+  ) {
+    mappingApiService.createPersonMapping(mapping).takeIf { it.isError }?.also {
+      with(it.errorResponse!!.moreInfo) {
+        telemetryClient.trackEvent(
+          "from-nomis-synch-contactperson-duplicate",
+          mapOf(
+            "existingNomisPersonId" to existing.nomisId,
+            "existingDpsContactId" to existing.dpsId,
+            "duplicateNomisPersonId" to duplicate.nomisId,
+            "duplicateDpsContactId" to duplicate.dpsId,
+            "type" to "PERSON",
```

Comment:
I intend to use the same event for person/contact duplicates hence the notion of "type"